Horatio/Services/ServiceRequestOperation.swift

```diff
@@ -110,6 +110,25 @@ open class DownloadServiceResponseOperation: GroupOperation {
     }
 }
 
+open struct ProcessServiceResponseErrors {
+    enum ErrorTypes: Error {
+        case notProcessed
+    }
+    
+    static let domain = "ProcessServiceResponseErrors"
+    
+    struct Codes {
+        static let unProcessed = 0
```

Comment:
Mismatched names: "unProcessed" and "notProcessed" to mean the same thing. "notProcessed" is the better choice.